paasta_tools/kubernetes_tools.py

```diff
@@ -1036,7 +1036,7 @@ def format_kubernetes_app(self) -> Union[V1Deployment, V1StatefulSet]:
             docker_url = self.get_docker_url()
             code_sha = get_code_sha_from_dockerurl(docker_url)
             complete_config: Union[V1StatefulSet, V1Deployment]
-            if self.get_persistent_volumes():
+            if self.get_persistent_volumes() or self.get_aws_ebs_volumes():
```

Comment:
Please update the tests as well: https://github.com/Yelp/paasta/blob/master/tests/test_kubernetes_tools.py#L800

Reply:
Updated tests. I have fixed the two failed tests in another PR. 
The three itests are failed for unknown reason (broken dependencies?).